{
  "Header": {
    "Original": "CVE-2019-16723 (In Cacti through 1.2.6, authenticated users may bypass authorization c ...)",
    "Line": 271480,
    "ID": "CVE-2019-16723",
    "Description": "(In Cacti through 1.2.6, authenticated users may bypass authorization c ...)"
  },
  "Annotations": [
    {
      "Original": "- cacti 1.2.7+ds1-1 (bug #941036)",
      "Line": 271481,
      "Type": "package",
      "Package": "cacti",
      "Kind": "fixed",
      "Version": "1.2.7+ds1-1",
      "BugNo": 941036
    },
    {
      "Original": "[buster] - cacti 1.2.2+ds1-2+deb10u2",
      "Line": 271482,
      "Type": "package",
      "Release": "buster",
      "Package": "cacti",
      "Kind": "fixed",
      "Version": "1.2.2+ds1-2+deb10u2"
    },
    {
      "Original": "[stretch] - cacti \u003cnot-affected\u003e (vulnerability introduced later)",
      "Line": 271483,
      "Type": "package",
      "Release": "stretch",
      "Package": "cacti",
      "Kind": "not-affected",
      "Description": "vulnerability introduced later"
    },
    {
      "Original": "[jessie] - cacti \u003cnot-affected\u003e (vulnerability introduced later)",
      "Line": 271484,
      "Type": "package",
      "Release": "jessie",
      "Package": "cacti",
      "Kind": "not-affected",
      "Description": "vulnerability introduced later"
    },
    {
      "Original": "NOTE: vulnerability introduced in",
      "Line": 271485,
      "Type": "NOTE",
      "Description": "vulnerability introduced in"
    },
    {
      "Original": "NOTE: https://github.com/Cacti/cacti/commit/cf73ae1a9f65b5a27d7f9d10c8e14835c3a76326",
      "Line": 271486,
      "Type": "NOTE",
      "Description": "https://github.com/Cacti/cacti/commit/cf73ae1a9f65b5a27d7f9d10c8e14835c3a76326"
    },
    {
      "Original": "NOTE: see Debian bug report for more information",
      "Line": 271487,
      "Type": "NOTE",
      "Description": "see Debian bug report for more information"
    },
    {
      "Original": "NOTE: https://github.com/Cacti/cacti/issues/2964",
      "Line": 271488,
      "Type": "NOTE",
      "Description": "https://github.com/Cacti/cacti/issues/2964"
    },
    {
      "Original": "NOTE: https://github.com/Cacti/cacti/commit/7a6a17252a1cbda180b61fff244cb3ce797d5264",
      "Line": 271489,
      "Type": "NOTE",
      "Description": "https://github.com/Cacti/cacti/commit/7a6a17252a1cbda180b61fff244cb3ce797d5264"
    },
    {
      "Original": "NOTE: https://github.com/Cacti/cacti/commit/c7cf4a26e4848872b48094e67f8d0a01dd7613d2",
      "Line": 271490,
      "Type": "NOTE",
      "Description": "https://github.com/Cacti/cacti/commit/c7cf4a26e4848872b48094e67f8d0a01dd7613d2"
    },
    {
      "Original": "NOTE: after further discussion, upstream issued a new fix which reverts previous commits",
      "Line": 271491,
      "Type": "NOTE",
      "Description": "after further discussion, upstream issued a new fix which reverts previous commits"
    },
    {
      "Original": "NOTE: https://github.com/Cacti/cacti/commit/cfb0733597af97abc92270de4f47cbfa32f9ce8b",
      "Line": 271492,
      "Type": "NOTE",
      "Description": "https://github.com/Cacti/cacti/commit/cfb0733597af97abc92270de4f47cbfa32f9ce8b"
    },
    {
      "Original": "NOTE: which turned out to be insufficient to fix the issue, follow up patches:",
      "Line": 271493,
      "Type": "NOTE",
      "Description": "which turned out to be insufficient to fix the issue, follow up patches:"
    },
    {
      "Original": "NOTE: https://github.com/Cacti/cacti/commit/9a1d2ec46d2dde23826c134ca70a0cd3bef43ee7",
      "Line": 271494,
      "Type": "NOTE",
      "Description": "https://github.com/Cacti/cacti/commit/9a1d2ec46d2dde23826c134ca70a0cd3bef43ee7"
    },
    {
      "Original": "NOTE: https://github.com/Cacti/cacti/commit/d5f98679a06aa96adfe04f60908f9108cfc9f7f7",
      "Line": 271495,
      "Type": "NOTE",
      "Description": "https://github.com/Cacti/cacti/commit/d5f98679a06aa96adfe04f60908f9108cfc9f7f7"
    },
    {
      "Original": "NOTE: https://github.com/Cacti/cacti/commit/4cecb19f6be8b84fa1c7b6450b66176007cb53df",
      "Line": 271496,
      "Type": "NOTE",
      "Description": "https://github.com/Cacti/cacti/commit/4cecb19f6be8b84fa1c7b6450b66176007cb53df"
    },
    {
      "Original": "NOTE: The original issue mentions only a bypass via graph_json.php but there are",
      "Line": 271497,
      "Type": "NOTE",
      "Description": "The original issue mentions only a bypass via graph_json.php but there are"
    },
    {
      "Original": "NOTE: additional permission checks missed while checking the issue fixed with the",
      "Line": 271498,
      "Type": "NOTE",
      "Description": "additional permission checks missed while checking the issue fixed with the"
    },
    {
      "Original": "NOTE: upstream commits.",
      "Line": 271499,
      "Type": "NOTE",
      "Description": "upstream commits."
    }
  ]
}